account_chatter/models/account_account.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2020 Vauxoo
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo import models, fields
+
+
+class AccountAccount(models.Model):
+
+    _name = "account.account"
+    _inherit = ['account.account', 'mail.thread']
+    _translate = False
+
+    name = fields.Char(track_visibility=True)
+    currency_id = fields.Many2one(track_visibility=True)
+    code = fields.Char(track_visibility=True)
+    deprecated = fields.Boolean(track_visibility=True)
+    reconcile = fields.Boolean(track_visibility=True)
+    user_type_id = fields.Many2one(track_visibility=True)
+    tax_ids = fields.Many2many(track_visibility=True)
+    tag_ids = fields.Many2many(track_visibility=True)
+    group_id = fields.Many2one(track_visibility=True)
+    realizable_account = fields.Boolean(track_visibility=True)
+
+    """
+    Funtion:
+        Track changes in tag_ids and tax_ids fields.
```

Comment:
This is done manually because fields tracking is not natively supported for many2many fields